### PR TITLE
chore: bump @rnx-kit/eslint-plugin to 0.2.0

### PR DESCRIPTION
**Pull request by tido64**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Bumps @rnx-kit/eslint-plugin to 0.2.0. This includes a fixer for the no-export-all rule. The rule will be treated as an error in a separate PR.

### Verification

`yarn lint` should still pass.

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Review by Saadnajmi — APPROVED** [review]

**Comment by tido64** [comment]
Oops, it looks like I accidentally wrote some code that doesn't work on Node 12. Is there any reason for staying on 12? Can we bump to 14?

**Comment by tido64** [comment]
Waiting for this fix to land before merging: https://github.com/microsoft/rnx-kit/pull/810

**Pull request merged**
